vulcanai/__init__.py

```diff
@@ -14,3 +20,19 @@
                           defaults={'logfilename': log_output_path})
 
 logger = logging.getLogger(__name__)
+
+
+def random_seeding(seed_value):
```

Comment:
should we call it something like `set_global_seed()`?

Reply:
sure, that's better!